src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Sorts all persons in the address book and lists them to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_SUCCESS = "Sorted all persons";
```

Comment:
Should this perhaps indicate the category the command sorts by? Perhaps "Sorted all persons by [category]"?